docs/architecture/ARCHITECTURE.md

```
@@ -113,10 +113,12 @@ Project Manager is a local-first ticket management system designed to enable eff
 - File-based templates
 
 **AI Integration**
-- Model Context Protocol (MCP) server
+- Model Context Protocol (MCP) server (mandatory standard)
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)